frontend/components/base/BaseCode.vue

```diff
@@ -60,6 +69,16 @@ export default {
     color: #3ef070;
     font-weight: bold;
   }
+  .hljs-deletion,
+  .hljs-number,
+  .hljs-quote,
+  .hljs-selector-class,
+  .hljs-selector-id,
+  .hljs-string,
+  .hljs-template-tag,
+  .hljs-type {
+    color: #febf96;
+  }
```

Comment:
change hljs by something more meaningfull

Reply:
`hljs` is the highlighting library injected class, I don't know if I can override

Reply:
Oh ok. If it's front another librairy better to not change then (but could be nice to add a comment on top // hljs is a class which came from the highlight librairy)